api/lib/mno_enterprise/concerns/controllers/jpi/v1/user_access_requests_controller.rb

```diff
@@ -0,0 +1,49 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::UserAccessRequestsController
+  extend ActiveSupport::Concern
+
+  #==================================================================
+  # Included methods
+  #==================================================================
+  # 'included do' causes the included code to be evaluated in the
+  # context where it is included rather than being executed in the module's context
+  included do
+    respond_to :json
+  end
+
+  # GET /mnoe/jpi/v1/user_access_requests
+  def index
+    @user_access_requests = MnoEnterprise::UserAccessRequest.active_requested(current_user.id)
+  end
+
+  # PUT /mnoe/jpi/v1/user_access_requests/:id/deny
+  def deny
+    return render_not_found('user_access_request') unless user_access_request
+    @user_access_request = user_access_request.deny.first
+    if @user_access_request.errors.empty?
+      @user_access_request = @user_access_request.load_required(:requester)
+      MnoEnterprise::EventLogger.info('access_denied', current_user.id, 'Access denied', @user_access_request)
+      MnoEnterprise::SystemNotificationMailer.access_denied(user_access_request.id).deliver_later
+      render 'show'
+    else
+      render_bad_request('deny', @user_access_request.errors)
```

Comment:
Have you checked if this work?
I had to do `object.errors.full_messages` on my controller